specification/src/main/asciidoc/jakarta-concurrency.adoc

```diff
@@ -2032,14 +1873,14 @@ enabled (this is the default behavior).
 [arabic]
 . A task instance that starts a transaction must complete the
 transaction before starting a new transaction.
-. The task provider uses the javax.transaction.UserTransaction interface
+. The task provider uses the `jakarta.transaction.UserTransaction` interface
 to demarcate transactions.
-. Transactions are demarcated using the begin(), commit() and rollback()
-methods of the UserTransaction interface.
+. Transactions are demarcated using the `begin()`, `commit()` and `rollback()`
+methods of the `UserTransaction` interface.
 . While an instance is in an active transaction, resource-specific
 transaction demarcation APIs must not be used (e.g. if a
-javax.sql.Connection is enlisted in the transaction instance, the
-Connection.commit() and Connection.rollback() methods must not be used).
+`javax.sql.Connection` is enlisted in the transaction instance, the
```

Comment:
Another occurrence of incorrect package name (already present in the old spec). Should be java.sql.Connection